handbook/product/tracking_issues.md

```diff
@@ -0,0 +1,88 @@
+# Tracking issues
```

Comment:
I am wondering if this should live under engineering instead of product.

If it is under product, it might feel like this is a process imposed on engineering by another function, but I don't think that is true. We do this because it is an effective way to communicate and plan engineering work to relevant stakeholders (the primary one being product).

I don't feel strongly about this but wanted to see what others thought: @christinaforney @lguychard

Reply:
I think that's a great point to communicate. Where it lives matters less to me than that this is also written down. Just because a process is in engineering it could be mandated by another team. If instead we clearly communicate why we use the tool and how it serves the team, we resolve this concern.

Reply:
> Where it lives matters less to me than that this is also written down.

+1